src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -288,6 +288,17 @@ private XmlNodeSyntax ParseXmlElement()
                 _pool.Free(attrs);
             }
         }
+
+        private static bool MatchingXmlNames(XmlNameSyntax name, XmlNameSyntax endName)
+        {
+            // NOTE we do not compare colon tokens.
+            //      as long as name and prefix are matching we will consider names equivalent
+            //      it would be a different error if colon are missing
+            return name.FullWidth == endName.FullWidth &&
```

Comment:
Is the logic now exactly equivalent to name.ToString() == endName.ToString()? If yes, please add this to the comment. If not, please explain what is the difference?